### ISSUE TITLE
Can't add existing pad - pad name not accepted

### ISSUE THREAD

**Issue by eternalbit** [mikifus/padland#46]
Created this pad: https://etherpad.wikimedia.org/p/thisisonlysomerandomtestcase

When I try to add the pad to Padland (note: I've added the wikimedia Server to my server list) by entering _thisisonlysomerandomtestcase_ in the field _Pad name_ and press _Create or go to_ I get this message _**The pad name contains invalid characters**_ and the pad is not opened.

Is the name maybe too long?

**Comment by mikifus** [comment]
I just tried that same pad you mentions. It works. Maybe you added a space at the end without noticing?

**Comment by eternalbit** [comment]
> Maybe you added a space at the end without noticing?

I checked again, no unintended space at the end.
I entered **_thisisonlysomerandomtestcase_** and got the message mentioned above.

My server settings:
name **_wikimedia_**
url **_https://etherpad.wikimedia.org_**
Etherpad Lite **_checked_**
url bad prefix **_/p/_**
jquery-plugin supported **_checked_**

Actually I tried omitting the **_/p/_** ... which led to the pad being added to my list. But when padland tries to open it I get an error message net::ERR_NAME_NOT_RESOLVED and clearly the url is bad: https://etherpad.wikimedia.orgthisis...

**Comment by mikifus** [comment]
I tried again. I pasted exactly the data you provided. Still can't reproduce it.

The error happens when the whole URL is not valid:
https://github.com/mikifus/padland/blob/6967659657304f727a8d1e865cf9216114a4c1bd/app/src/main/java/com/mikifus/padland/NewPadActivity.java#L194

This means that the error can be on your server definition, not on the pad name. I insist on the white space as the Android text fields make the keyboard add spaces automatically.
